### PR TITLE
chore(wing): update queries from upstream

### PR DESCRIPTION
**Pull request by lucario387**

### CODE REVIEW

**Review by clason — APPROVED** [review]

**Comment by lucario387** [comment]
@clason I also cherry pick the github PR into here as well.

File: lockfile.json [hunk 1]
```diff
@@ -324,7 +324,7 @@
     "revision": "8a841fb20ce683bfbb3469e6ba67f2851cfdf94a"
   },
   "liquidsoap": {
-    "revision": "bbef4df4dc5b324455ad1ea4770bbed0df5130ea"
+    "revision": "4acf9035d852fc65e013f05f597635204e6d1ca5"
```
Comment (clason) [inline lockfile.json:327]: better leave that for the automated PR; for bisecting purposes, it's not a good hide to "hide" unrelated changes -- if it says "wing" on the tin, wing is all that should be in it ;)

Comment (lucario387) [reply lockfile.json:327]: Im out for some hours. I'm certain the wing changes will resolve the issue

Comment (clason) [reply lockfile.json:327]: There's no hurry; people can live without the old wing (and liquidsoap) parser for a few hours more ;)
